Plot window fix

jcmkang opened the pull request:
Hello there!

I combined the Box Plot Window and Bar Chart Window together to reduce the number of windows displayed to the user. (hard to keep track of all the windows when user is making multiple selection of countries..

I added 
1). simple analysis on the bottom of the chart. 
2) close button on bottom of page.

I am open to adding/adjusting some more analysis. 
Also open to hearing about placement.. I thought maybe having the analysis as a side panel with labels going top to bottom!

I didn't do resizing on this and put it as False rather..

 lmk what you think!

Commit:
Combined Box Plot Window and Bar Chart Window together.. Added analysis and close button on bottom of page.

Commit:
Fixed 'listing' of the selected countries

morosebose commented:
James, this looks great! I'm very impressed. I have a few suggestions/questions. Only the first is one I feel pretty sure about. The others are minor, quibbles really.

- I was wrong about where the Analysis box goes. Since the data there is from the box plot, it should go with the box plot, and so having it on top of the box plot would make more sense instead of having it separated from the box plot by the bar chart. What do you think?
- I'm not a fan of the rotated xticks, tbh. I'd not mind them as much if they ended precisely where the tick meets the plot outine, but I think they look sloppy when the figures are on either side of that line. 
- The xticks/yticks are a bit too small for my comfort. Maybe .5pt or 1pt larger? If it's a question of either this or the non-rotated xticks, I'd pick the non-rotated xticks.
- `Plot and Analysis` rather than `Plot & Analysis` for the window title?
- Leave the word `Countries:` out of the header to the analysis window, just list the countries?
- `1st quartile` and `3rd quartile` instead of `Lower q1` and `Upper q3`?
- List the data in this order: Total, Min, 1st Quartile, Median, 3rd Quartile, Max.

If you disagree with any of these suggestions, feel free to stick with what you have rn! I'll defer to you and sign off. To reiterate, I love this, and it's super cool! I'm very excited. The `Close` button is a great touch!

morosebose reviewed (changes requested):
Looks great, I've requested a few minor tweaks under `Conversation`!

Commit:
Changed Plot n Analysis layout and total num countries selected label

jcmkang commented:
Hey there!

I made some adjustments to the PlotNAnalysis Window, listbox window as well!

- analysis box went up above the box chart. Makes more sense. 
	- I added some labels for all the analysis. km^2 for area, and “blank” for population… but should we add persons? People? Leave it blank? up to suggestions!

- changed the listing of the data in order of Total, Min, 1st Quartile, Median, 3rd Quartile, Max...

- created interactive label for when user chooses the number of countries from the listbox 

- changed the sticks to have bins.. I put it at 6 for now.. if not, w/o the rotation, the text would run to the next tick. 


of course always open to suggestions and tweeks
Let me know what ya think!!

Commit:
minor change
